README.md

```diff
@@ -1,2 +1,168 @@
-# ces-js-parser
-The js library for parsing Casper Event Standard events
+# JS CES Parser
+
+`ces-js-parser` parses contract-level events that follow
+the [Casper Event Standard](https://github.com/make-software/casper-event-standard).
+
+The library is built on top of the 'casper-js-sdk' and operates on types defined by the SDK.
+
+## Install
+
+``
+npm install --save ces-js-parser
+``
+
+## Usage
+
+Here is an example of parsing CES events using `ces-js-parser` from a real Testnet deploy loaded
+with `casper-js-sdk`:
+
+```typescript
+(async () => {
```

Comment:
Wrapping examples into an anonymous function is redundant, however it would be nice to have the import example instead.